lib/dry/web/roda/templates/monitor.rb.tt

```diff
@@ -1,6 +1,7 @@
 <%= config[:camel_cased_app_name] %>::Container.boot :monitor do
   init do
     require "dry/monitor"
+    Dry::Monitor.load_extensions(:sql)
```

Comment:
@timriley I wonder if this is worth a separate PR? It changes functionality rather than just making it work.

Also it makes the generator tests fail:

```ruby
NoMethodError: undefined method `env' for #<#<Class:0x00005563a465aad8>:0x00005563a465a6a0>
```

Reply:
Yeah, fair enough. Happy to defer this.

Reply:
:+1: